cnxepub/models.py

```diff
@@ -467,6 +486,58 @@ def references(self):
             return []
         return self._references
 
+    def fix_html_ids(self, mapping):
```

Comment:
The presence of IDs on specific tags is a feature of of HTML5 dialect, so arguably is part of our model specification. AS you point out, it's hard to link without them. :-)